src/test/java/racingcar/CarTest.java

```diff
@@ -0,0 +1,61 @@
+package racingcar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class CarTest {
+    private SimpleCar car;
+
+    @BeforeEach
+    void init() {
+        car = new SimpleCar("test");
+    }
```

Comment:
```suggestion
    private SimpleCar car = new SimpleCar("test");
```

각각의 테스트는 분리되어 동작하기 때문에 BeforeEach 없이 위와 같이 할당해 사용하셔도 괜찮을 것 같아요. BeforeEach를 써도 무관하나 코드 양을 줄일 수 있습니다.